src/screens/ImageBase.tsx

```diff
@@ -45,41 +91,23 @@ const ImageBase: React.FC = () => {
     },
     imageContainer: {
       backgroundColor: "#E5E5E5",
-      margin: 20,
-      borderRadius: 20,
-      height: 300,
-      width: 300,
+      height: 100,
+      width: 100,
+      margin: 5,
     },
-
     imageListContainer: {
-      height: "55%",
+      height: "65%",
       width: "90%",
       justifyContent: "center",
-      alignItems: "center",
       marginTop: 30,
+      alignItems: "center",
     },
-    heading: {
-      fontSize: 25,
-      fontFamily: "poppinsExtraBold",
-      position: "absolute",
-      top: "7%",
-      alignSelf: "center",
-    },
-    pexels: {
-      height: 40,
-      width: 40,
-    },
-    pexelsContainer: {
-      position: "absolute",
-      bottom: "5%",
-      left: "7%",
+    _imageListContainerStyle: {
```

Comment:
Perhaps rename to a useful name 

Thanks!

Reply:
Fixed in [1d3f951](https://github.com/Chillandchat/mobile/pull/84/commits/1d3f95115d2956f6f3368bf22b7d1f1e8141e645)